casper-feed/src/main/proto/admin.proto

```
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package casper.feed;
```

Buf (1.55.1)

Files with package "casper.feed" must be within a directory "casper/feed" relative to root but were in directory "casper-feed/src/main/proto".

(PACKAGE_DIRECTORY_MATCH)